R/render.R

```diff
@@ -87,7 +87,7 @@ render_pdf <- function(x,
                              output_file = filename,
                              output_dir = display_loc,
                              output_options = list(keep_tex = keep_tex),
-                             params = list(x = x$display,
+                             params = list(x = x,
                                            header = hf_process(x$header, escape_latex = escape_latex),
```

Comment:
Since we're passing the full obj, we no longer need the individual params, I think. So it can be similar to the qmd. I will just make an issue to address later.